prow/plugins/label/label_test.go

```diff
@@ -480,6 +490,11 @@ func TestLabel(t *testing.T) {
 		if len(fakeClient.IssueCommentsAdded) == 0 && tc.expectedBotComment {
 			t.Error("expected a bot comment but got none")
 		}
+		if tc.expectedBotComment && len(tc.expectedCommentText) > 0 {
+			if len(fakeClient.IssueComments) != 1 || strings.Index(fakeClient.IssueComments[1][0].Body, tc.expectedCommentText) == -1 {
+				t.Errorf("%v", fakeClient.IssueComments[1][0].Body)
```

Comment:
Please emit both the actual and expected values.

Reply:
Thanks again!